app/sprinkles/core/src/Util/CheckEnvironment.php

```diff
@@ -125,6 +125,12 @@ public function checkAll()
             $problemsFound = true;
         }
 
+        if ($this->checkDirectories()) {
+            $problemsFound = true;
+        // Skip checkPermissions() if the required directories do not exist.
+        return $problemsFound;
```

Comment:
Careful with the indentation here

Reply:
Thanks for the information! I will look over the PSR guides and get more familiar with the UF style guide as well.